ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Cluster/CVEsTable.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { gql } from '@apollo/client';
+import { Text } from '@patternfly/react-core';
+import { Table, Thead, Tr, Th, Td, ExpandableRowContent, Tbody } from '@patternfly/react-table';
+
+import CvssFormatted from 'Components/CvssFormatted';
+import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
+import VulnerabilityFixableIconText from 'Components/PatternFly/IconText/VulnerabilityFixableIconText';
+import { TableUIState } from 'utils/getTableUIState';
+import useSet from 'hooks/useSet';
+
+import PartialCVEDataAlert from '../../components/PartialCVEDataAlert';
+import { getPlatformEntityPagePath } from '../../utils/searchUtils';
+
+function displayCveType(cveType: string): string {
+    switch (cveType) {
+        case 'K8S_CVE':
+            return 'Kubernetes';
+        case 'ISTIO_CVE':
+            return 'Istio';
+        case 'OPENSHIFT_CVE':
+            return 'Openshift';
+        default:
+            return cveType;
+    }
+}
+
```

Comment:
@charmik-redhat 

This function is used to convert Cluster CVE type enum into a display version for the user. These values appear to come from here: https://github.com/stackrox/stackrox/blob/master/proto/storage/cve.proto#L30.

Is there a subset we can expect to receive for **Cluster CVEs** that need to be handled here? `K8S` and `OPENSHIFT` are the obvious ones, but do we need to handle the others?

Reply:
Will ask at today's sync and follow up in a later PR if necessary.